_includes/footer_custom.html

```diff
@@ -1,3 +1,6 @@
-{%- if site.footer_content -%}
-<p class="text-small text-grey-dk-100 mb-0">{{ site.footer_content }}</p>
-{%- endif -%}
+<p class="text-small text-grey-dk-100 mb-0">
+  &copy; 2022 Memo Bank. Publié sous license
+  <a href="https://github.com/memobank/growth-guide/blob/main/LICENSE.txt"
+    >Creative Commons Attribution 4.0 International</a
+  >. Hébergé sur <a href="https://pages.github.com">GitHub Pages</a>.
```

Comment:
The tags are weirdly aligned here.
It might be worth installing [prettier](https://prettier.io) at some point.